crates/runtime/benches/setup/mod.rs

```diff
@@ -18,7 +18,7 @@ pub(crate) async fn setup_benchmark(
 
     let app = build_app(upload_results_dataset);
 
-    let rt = Runtime::new(Some(app), Arc::new(vec![])).await;
+    let rt = Runtime::new("spice".to_string(), Some(app), Arc::new(vec![])).await;
```

Comment:
Nit: we can have separate method to specify the name and default to "spice" or auto-generated uuid. This way we don't need to update all tests/benches.


let rt = Runtime::new()
rt.with_name(...)

Reply:
Thanks for pointing out 👍. Moved instance name generation inside runtime, we don't actually need to modify it outside.